controller/src/main/java/io/pravega/controller/server/SegmentHelper.java

```diff
@@ -248,26 +270,32 @@ public void processingFailure(Exception error) {
 
             @Override
             public void connectionDropped() {
+                log.warn("createTransaction {}/{}/{} connectionDropped", scope, stream, segmentNumber);
                 result.completeExceptionally(new WireCommandFailedException(type, WireCommandFailedException.Reason.ConnectionDropped));
             }
 
             @Override
             public void wrongHost(WireCommands.WrongHost wrongHost) {
+                log.warn("createTransaction {}/{}/{} wrong host", scope, stream, segmentNumber);
                 result.completeExceptionally(new WireCommandFailedException(type, WireCommandFailedException.Reason.UnknownHost));
             }
 
             @Override
             public void transactionCreated(WireCommands.TransactionCreated transactionCreated) {
+                log.info("createTransaction {}/{}/{} TransactionCreated", scope, stream, segmentNumber);
```

Comment:
If we are creating txns at a high rate, then are we going to be printing a lot of these INFO messages, it might be best to make this one DEBUG what do you think?

Reply:
agreed.